### PR TITLE
Update audit as of 2020-04-27

### PR DESCRIPTION
**Pull request by spiffxp**
This was intended to pick up the results of running scripts/terraform introduced in https://github.com/kubernetes/k8s.io/pull/806

It also picked up:
- removing some stray `projectView` permissions on `k8s-conform` buckets
- that time where I accidentally the `kubernetes-public` project-wide `ssh-keys` metadata
  - I was trying to manually delete an old key from `spiffxp-node-e2e-project` but forgot the `--project` flag and was defaulted to `kubernetes-public`
  - I would restore the keys but they're listed as expiring on `2020-01-28` 

/cc @dims 
to ack the ssh-keys removal
/cc @thockin @bartsmykla @cblecker 
whomever is interested in reviewing the audit

### CODE REVIEW

File: audit/projects/kubernetes-public/service-accounts/boskos-janitor-test@kubernetes-public.iam.gserviceaccount.com/iam.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "bindings": [
+    {
+      "members": [
+        "serviceAccount:kubernetes-public.svc.id.goog[test-pods/boskos-janitor]"
```
Comment (thockin) [inline audit/projects/kubernetes-public/service-accounts/boskos-janitor-test@kubernetes-public.iam.gserviceaccount.com/iam.json:5]: Why is it still called "test-pods"

File: audit/projects/kubernetes-public/services/compute/project-info.json [hunk 2]
```diff
@@ -1,22 +1,22 @@
 {
   "commonInstanceMetadata": {
-    "fingerprint": "9_em4FUS_lU=",
```
Comment (thockin) [inline audit/projects/kubernetes-public/services/compute/project-info.json:3]: Another candidate for being removed  @bartsmykla

File: audit/projects/spiffxp-boskos-project-01/description.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "createTime": "2020-04-26T17:09:06.114Z",
+  "lifecycleState": "ACTIVE",
+  "name": "spiffxp-boskos-project-01",
```
Comment (thockin) [inline audit/projects/spiffxp-boskos-project-01/description.json:4]: Let's not forget to clean these up :)

Comment (spiffxp) [reply audit/projects/spiffxp-boskos-project-01/description.json:4]: this is exactly why I put my name on them

File: audit/projects/kubernetes-public/service-accounts/boskos-janitor-test@kubernetes-public.iam.gserviceaccount.com/iam.json (hunk 1 above)
Comment (spiffxp) [reply audit/projects/kubernetes-public/service-accounts/boskos-janitor-test@kubernetes-public.iam.gserviceaccount.com/iam.json:5]: See https://github.com/kubernetes/k8s.io/pull/806#discussion_r416215390

File: audit/projects/kubernetes-public/services/compute/project-info.json (hunk 2 above)
Comment (bartsmykla) [reply audit/projects/kubernetes-public/services/compute/project-info.json:3]: ack

Comment (bartsmykla) [reply audit/projects/kubernetes-public/services/compute/project-info.json:3]: Done: #809

**Comment by spiffxp** [comment]
/close
this is out of date, I'll open another audit PR tomorrow

**Pull request closed without merging**